readthedocs/templates/projects/onboard_detail.html

```diff
@@ -60,11 +60,14 @@ <h2>{% trans "Your documentation failed to build" %}</h2>
         {% endif %}
       {% else %}
         <h2>{% trans "Your documentation is building" %}</h2>
-
+        
+        {% url "builds_detail" project_slug=project.slug build_pk=onboard.build.pk as latest_build_url %}
         <p>
         {% blocktrans trimmed %}
           You'll be able to view your documentation in a minute or two,
           once your project is done building.
+          For more information about the build,
+          click <a href="{{ latest_build_url }}">here.</a>
```

Comment:
We can just put the link in the above text `Your documentation is building`

Reply:
Yes, that sounds better. I've made the change.